### PR TITLE
feat: Add priorityClassName

### PR DESCRIPTION
**Pull request by ajalewis**
Hi,

I rely on pms a lot in my local k8s lab and consider it to take precedence over other services I have running so wanted to add priorityClassName.

### CODE REVIEW

File: charts/plex-media-server/templates/statefulset.yaml [hunk 1]
```diff
@@ -26,6 +26,9 @@ spec:
         {{- toYaml .Values.nodeSelector | nindent 8 }}
       affinity:
         {{- toYaml .Values.affinity | nindent 8 }}
+      {{- if .Values.priorityClassName }}
+      priorityClassName: "{{ .Values.priorityClassName }}"
```
Comment (cilindrox) [inline charts/plex-media-server/templates/statefulset.yaml:30]: nit: preference is to pipe to `quote`, eg:
```suggestion
      priorityClassName: {{ .Values.priorityClassName | quote }}
```

**Review by cilindrox — APPROVED** [review]
LGTM 👍 - thanks @ajalewis!

LMK if you'd like to use the `quote` func, otherwise we can update the template later.

**Comment by ajalewis** [comment]
@cilindrox Have changed this as per your suggestion. Thanks!

**Comment by cilindrox** [comment]
Thanks for the contribution @ajalewis ! merging and shipping this as part of v0.2.0

**Review by MarshallAsch — APPROVED** [review]

**Pull request merged**